variable/technology/power-plant.yaml

```diff
@@ -0,0 +1,799 @@
+   description: CO2 emmissions of a typical new biomass power plant
+                biomass from dedicated energy crops and biomass imports.
+   unit: tons/MWh
+
+CO2 Emmissions|Electricity|Nuclear:
+   description: CO2 emmissions of a typical nuclear power plant
+   unit: tons/MWh
+
+CO2 Emmissions|Electricity|Gas|OCGT:
+   description: CO2 emmissions of a typical Open Cycle Gas Turbine power plant
+   unit: tons/MWh
+
+CO2 Emmissions|Electricity|Gas|Coal:
+   description: CO2 emmissions of a typical coal power plant
+   unit: tons/MWh
+
+CO2 Emmissions|Electricity|Lignite:
+   description: CO2 emmissions of a typical Lignite power plant
+   unit: tons/MWh
+
+# Variables related to the production cost of a power plant
+# Note that:
+# linear   term variable cost   [€/GWh] = Linear Variable Cost [Mcal/GWh] * Price|Primary Energy [€/Mcal] + Operation and Maintenance Variable Cost [€/MWh] * 1e-3
+# constant term variable cost   [€/h]   = Variable Cost (Intercept) [Mcal/h] * Price|Primary Energy [€/Mcal]
+Variable Cost (Intercept)|Electricity|Coal:
+   description: Constant term (intercept) of the heat rate straight line. Cost of the power generation unit who belongs to the specified power
+                plant category.
+   unit: Mcal/h
+
+Variable Cost (Intercept)|Electricity|Gas:
+   description: Constant term (intercept) of the heat rate straight line. Cost of the power generation unit who belongs to the specified power
+                plant category.
+   unit: Mcal/h
+
+Variable Cost (Intercept)|Electricity|Nuclear:
+   description: Constant term (intercept) of the heat rate straight line. Cost of the power generation unit who belongs to the specified power
+                plant category.
+   unit: Mcal/h
+
+Variable Cost (Intercept)|Electricity|Oil:
+   description: Constant term (intercept) of the heat rate straight line. Cost of the power generation unit who belongs to the specified power
+                plant category.
+   unit: Mcal/h
+
+Variable Cost (Slope)|Electricity|Coal:
+   description: Linear term (slope) of the heat rate straight line. Cost of the power generation unit who belongs to the specified power
+                plant category.
+   unit: Mcal/GWh
+
+Variable Cost (Slope)|Electricity|Gas:
+   description: Linear term (slope) of the heat rate straight line. Cost of the power generation unit who belongs to the specified power
+                plant category.
+   unit: Mcal/GWh
+
+Variable Cost (Slope)|Electricity|Nuclear:
+   description: Linear term (slope) of the heat rate straight line. Cost of the power generation unit who belongs to the specified power
+                plant category.
+   unit: Mcal/GWh
+
+Variable Cost (Slope)|Electricity|Oil:
+   description: Linear term (slope) of the heat rate straight line. Cost of the power generation unit who belongs to the specified power
+                plant category.
+   unit: Mcal/GWh
+
+Operation and Maintenance Variable Cost|Electricity|Coal:
```

Comment:
I find this a bit confusing between 
 - `Variable Cost`, which has an intercept and a slope,
 - `Operation and Maintenance Variable Cost`, which is a constant - so is it a variable cost? is it related to the capacity or the generation level?

Reply:
Yeah, for:
- Variable Cost| Slope [€/GWh]: It refers to the production cost of each energy unit (GWh, MWh,etc.).
- Variable Cost| Intercept [€/GWh]: It refers to the unit commitment variable of each power plant per unit time. 

I renamed the O&M cost to `Operation and Maintenance Cost` [€/MWh]. It refers to the O&M cost that must be assumed in the production cost of the power plant and is valued for each energy unit produced.